apps/fabric-website/src/components/Nav/Nav.CollapsibleSectionTitleStyles.tsx

```diff
@@ -0,0 +1,43 @@
+import {
+  ICollapsibleSectionTitleProps,
+  ICollapsibleSectionTitleStyles
+} from '@uifabric/experiments/lib/CollapsibleSection';
+import { ITheme, getFocusStyle } from 'office-ui-fabric-react';
+
+export const getStyles = (props: ICollapsibleSectionTitleProps & { theme: ITheme }): ICollapsibleSectionTitleStyles => {
+  const { theme } = props;
+
+  return {
+    root: [
+      getFocusStyle(theme),
+      {
+        width: '100%',
+        display: 'flex',
+        alignItems: 'center',
+        background: 'none',
+        border: 'none',
+        color: theme.palette.neutralQuaternaryAlt,
```

Comment:
I've heard somewhere in a different PR that this color is being deprecated - but I don't know the replacement... @Jahnp maybe able to clarify?